scripts/check_environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+
+containers=$(docker ps --format "{{.ID}}" | wc -l)
+
+if [ "$containers" -eq "0" ]; then
+   exit 0
+fi
+echo "environment is not clean"
```

Comment:
Could we list the number of containers that are still running? or a least of them?

Reply:
@mdelapenya done